<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PathPicker Example</string>
  </property>
  <widget class="QSint::PathPicker" name="PathPicker">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Objects to pick</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbFile">
      <property name="text">
       <string>File to open</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbFiles">
      <property name="text">
       <string>Files to open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbDir">
      <property name="text">
       <string>Directory to open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbFileToSave">
      <property name="text">
       <string>File to save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>87</width>
     <height>73</height>
    </rect>
   </property>
   <property name="title">
    <string>Dialog look</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="rbSystemDialog">
      <property name="text">
       <string>System</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbQtDialog">
      <property name="text">
       <string>Qt based</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSint::PathPicker</class>
   <extends>QWidget</extends>
   <header>QSint</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
